manifest-spec.md

```diff
@@ -0,0 +1,37 @@
+# Package Manifests
+
+
+## Assumptions
+
+The package manifest specification makes the following assumptions.
+
+1. Package manifests are primarily for non-contract actors and will not be consumed by contracts themselves.
+2. Following from that, we will likely store the manifest on some distributed protocol like IPFS or Swarm (although this is jumping ahead in the discussion).
+3. Since our off chain actors are primarily software, the message format needs to be easily readable by most programming languages.
+4. That said, it's likely human actors will write the manifest, meaning it needs to be in a format humans can easily understand and comprehend.
+5. Contrasting to the above, we need a format that takes up the least amount of space possible so that it costs the least to make available
+
+
+## Data to be stored within the manifest
+
+- package name
```

Comment:
https://gitter.im/ethpm/Lobby?at=57f56bf03c59573f6f0fd448
> Tim Coulter @tcoulter Okt. 05 23:09
> I’m contemplating we remove the name. That sounds like it might be registry level - lest the manifest name and the registered name not match.

I support this 👍